### DISCUSSION TITLE
How to enable darkmode switch?

### DISCUSSION THREAD

**Discussion by html5syt** [squidfunk/mkdocs-material#7071] [Q&A] [answered]
![Screenshot_20240421_191220_mark via](https://github.com/squidfunk/mkdocs-material/assets/53525043/f428c6cb-8e5b-4cfe-97d1-ac3b3fab32b2)
I found all of the offical docs,but I did't found how to enable darkmode switch.
Who can tell me?

**Comment by martinbira** [comment, marked as answer]
It's right there in the docs: https://squidfunk.github.io/mkdocs-material/setup/changing-the-colors/#color-palette-toggle

**Reply by html5syt** [reply]
sorry...I'm careless...
But I was added it,but it was no switch now. This my config file:[https://github.com/html5syt/Always-Bravely-Docs/blob/main/mkdocs-conf.yml](https://github.com/html5syt/Always-Bravely-Docs/blob/main/mkdocs-conf.yml)
And it's my (imcomplete) website: [docs.bravely.pp.ua](docs.bravely.pp.ua).

I use github pages&actions to build my docs.

**Reply by kamilkrzyskow** [reply]
@html5syt 
You have a:
`theme -> palette -> theme -> medialist` structure 

This isn't what is in the documentation:
`theme -> palette -> medialist`

**Reply by html5syt** [reply]
感谢(^_^) 
我对mkdocs还不是太了解😂看来得多在Github上参考一下了
